hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/snapshot/OMSnapshotPurgeRequest.java

```diff
@@ -140,6 +142,7 @@ private void updateSnapshotInfoAndCache(SnapshotInfo snapInfo, OmMetadataManager
       // current snapshot is deleted. We can potentially
       // reclaim more keys in the next snapshot.
       snapInfo.setDeepClean(false);
+      snapInfo.setDeepCleanedDeletedDir(false);
```

Comment:
setting it to false makes the next and the next of next snapshots to perform deep cleaning.
I dunno but why is it use 'false' rather than 'true' to indicate performing deep cleaning? or maybe i didn't understand it.

Reply:
The flag indicates that the snapshot if the snapshot has already been deepCleaned or not. So when it is set to false deep cleaning would be performed on the snapshot